config/github-webhook/verify-signed-off.rb

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env ruby
+
+#
+# Simple(ish) Ruby Sinatra bot to verify that all commits on a Github
+# pull request have a Signed-off-by line in the commit message.
+#
+# Note that this code was heavily inspired by
+# http://git-scm.com/book/en/v2/GitHub-Scripting-GitHub
+#
+##################################################################################
+
+# Yum-installed gems; apparently CentOS doesn't automatically add them
+# to the ruby search path.  :-( There's probably a better way to do
+# this, but I'm not One with the Ruby...
+$LOAD_PATH.unshift '/usr/lib/ruby/gems/1.8/gems/crack-0.1.8/lib'
```

Comment:
It doesn't work with `irb` on flatbed, and it does on my own RHEL 6 VM, so something's up with the ruby setup on flatbed:
```
[ofiwg-bot@flatbed ~]$ irb
irb(main):001:0> require 'rubygems'
=> true
irb(main):002:0> require 'json'
LoadError: no such file to load -- json
        from /usr/lib/ruby/site_ruby/1.8/rubygems/custom_require.rb:31:in `gem_original_require'
        from /usr/lib/ruby/site_ruby/1.8/rubygems/custom_require.rb:31:in `require'
        from (irb):2
        from :0
```